edge-modules/azure-monitor/src/ExampleDeployment.json

```diff
@@ -0,0 +1,85 @@
+{
+  "modulesContent": {
+    "$edgeAgent": {
+      "properties.desired": {
+        "schemaVersion": "1.0",
+        "runtime": {
+          "type": "docker",
+          "settings": {
+            "minDockerVersion": "v1.25",
+            "loggingOptions": "",
+            "registryCredentials": {}
+          }
+        },
+        "systemModules": {
+          "edgeAgent": {
+            "type": "docker",
+            "settings": {
+              "image": "mcr.microsoft.com/azureiotedge-agent:1.1",
+              "createOptions": ""
+            }
+          },
+          "edgeHub": {
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "settings": {
+              "image": "mcr.microsoft.com/azureiotedge-hub:1.1",
+              "createOptions": "{\"HostConfig\":{\"PortBindings\":{\"443/tcp\":[{\"HostPort\":\"443\"}],\"5671/tcp\":[{\"HostPort\":\"5671\"}],\"8883/tcp\":[{\"HostPort\":\"8883\"}]}}}"
+            }
+          }
+        },
+        "modules": {
+          "AzureMonitorForIotEdgeModule": {
+            "settings": {
+                "image": "mcr.microsoft.com/azureiotedge-metrics-collector:0.1",
+                "createOptions": "{\"HostConfig\":{\"LogConfig\":{\"Type\":\"json-file\",\"Config\":{\"max-size\":\"4m\",\"max-file\":\"7\"}}}}"
+            },
+            "type": "docker",
+              "env": {
+                "UploadTarget": {
+                  "value": "AzureMonitor"
+                },
+                "LogAnalyticsWorkspaceId": {
+                    "value": "<log analytics workspace id>"
+                },
+                "LogAnalyticsSharedKey": {
+                    "value": "<log analytics workspace shared key>"
+                },
+                "MetricsEndpointsCSV": {
+                    "value": "http://edgeHub:9600/metrics,http://edgeAgent:9600/metrics"
+                },
+                "HubResourceID": {
+                    "value": "<IoT hub resource id>"
+                },
+                "ScrapeFrequencyInSecs": {
+                    "value": "300"
+                },
+                "AllowedMetrics": {
+                    "value": ""
+                },
+                "BlockedMetrics": {
+                    "value": ""
+                },
```

Comment:
If this is serving as an example it might be a good idea to include the format that this env var takes.

Reply:
Empty is the default value for this env. If the developer needs more details he/she can find them in the README.md file.